spec/models/package_manager/maven_spec.rb

```diff
@@ -108,7 +108,7 @@
       it "should return nil" do
         allow(described_class).to receive(:download_pom).and_raise(PackageManager::Maven::POMNotFound.new("https://a-spring-url"))
 
-        expect(described_class.mapping({group_id: "org", artifact_id: "foo", version: "1.0.0"})).to eq(nil)
```

Comment:
Rubocop auto-corrected a bunch of whitespaces, the only major changes here were adding the specs on https://github.com/librariesio/libraries.io/pull/2840/files#diff-06c2fce0a53ffddef834b0f150c296f813f1c482f19ad291130a115870ebebf7R205